priv/templates/extended_bin

```diff
@@ -263,8 +263,11 @@ add_path() {
 check_replace_os_vars() {
     IN_FILE_PATH=$(add_path $1 $2)
     OUT_FILE_PATH="$IN_FILE_PATH"
+    SRC_FILE_PATH="$IN_FILE_PATH.src"
     ORIG_FILE_PATH="$IN_FILE_PATH.orig"
-    if [ $RELX_REPLACE_OS_VARS ]; then
+    if [ -f "$SRC_FILE_PATH" ]; then
+        replace_os_vars "$SRC_FILE_PATH" "$OUT_FILE_PATH"
```

Comment:
Basically, it is assumed that if sys.config.src exists, sys.config gets overwritten every time, right? that seems fair. Will just need to make it clear that it behaves like other Erlang/OTP/rebar3 *.src files basically.